Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -7288,12 +7288,11 @@ void CPhysicalGeometry::SetBoundControlVolume(const CConfig* config, unsigned sh
      All nodes that are shared by multiple symmetries have to get a corrected normal. */
   static constexpr size_t MAXNSYMS = 100;
   unsigned short Syms[MAXNSYMS] = {0};
-
+  symmetryNormals.resize(nMarker);
```

Comment:
`symmetryNormals` is shared, you'll either want to resize it outside OpenMP parallel code or, if it has to be done here, safeguard it with
```
SU2_OMP_SAFE_GLOBAL_ACCESS(symmetryNormals.resize(nMarker);)
```

Reply:
Thanks Johannes, I was already wondering what I should do with it. I will move it outside of openmp code.

Reply:
btw does the above have a performance impact? I see for instance that for a similar routine in CMultiGridGeometry.cpp, the entire function is inside the SAFE_GLOBAL_ACCESS guard.

Reply:
The safeguarding involves two barriers and serialization (only the master thread executes the code). In this specific case, where serialization is a must, you basically save the synchronization overhead due to the barriers if you move the code out of the parallel region. In general, you'll also lose performance if you place something inside the safeguards that can be parallelized efficiently, since all threads except the master idle during the safeguarded block. `CMultiGridGeometry::SetBoundControlVolume` could probably be changed along the lines of `CPhysicalGeometry::SetBoundControlVolume`. How large is `nMarker` usually?

Reply:
nMarker here is just the number of marker/boundaries in the mesh, so that would be of O(10) usually. 
For every marker we store a vector with the ipoint and corrected normals when they are on nodes with shared symmetry or euler wall. This vector is zero when the marker is not symmetry/euler and also 0 when the symmetry/euler does not have nodes that are shared with other symmetry/eulerwalls.

Reply:
Hi @jblueh I removed the SAFE_GLOBAL_ACCESS from CMultiGridGeometry::SetBoundControlVolume and replaced it with SU2_OMP_FOR_DYN(1) for the individual loops over markers.